frontend/src/theme/components/Badge.ts

```diff
@@ -23,6 +23,9 @@ const Badge = {
     stage: {
       backgroundColor: "blue.50",
     },
+    role: {
```

Comment:
this seems redundant since stage is the same color, but as we scale we could want to use separate variants and possibly want to change their colors.

Reply:
stage/role do seem to serve the same purpose at the moment (i.e. is it translation/review) so i think keeping it the same color makes sense, but having a separate variant is reasonable (and also doesn't cause any issues anyway)
if we were to differentiate, i think it would be nice to have different colors for translator vs. reviewer instead, so users can more easily tell if (for example when viewing story cards) that is a story they are translating or reviewing

Reply:
good idea! will keep in mind